watertap/unit_models/electrodialysis_1D.py

```diff
@@ -195,6 +195,14 @@ class Electrodialysis1DData(InitializationMixin, UnitModelBlockData):
             description="The electrical operation mode. To be selected between Constant Current and Constant Voltage",
         ),
     )
+    CONFIG.declare(
+        "calculate_limiting_current_density",
```

Comment:
Wasn't involved in this discussion but any disadvantages to just have a "None" option for the limiting current density method (instead of a new config option)?

Reply:
Thanks for quick reviewing!   You mean combine this with another configuration?  I don't think that makes a technical difference.   But I would stay as-is because 1) this is clearer and 2) I don't think it's critical enough to forfeit the work I have done here.

Reply:
If you insist 22 vs 21 config is critical,  I will close this and redo in the future.  This ED unit is large and linked with mutiple fs/analyses and such minor change needs careful inspection in multiple places so it may need longer time than you would think.  I don't know if that is the right/necesary analogy to RO.  I still think this is trivial and not worthy of more time that could be used to move forward much analysis work down the way.